### PR TITLE
Add force to books, extra to space law

### PR DESCRIPTION
**Pull request by datlo**
Books now have 2 force, dealing light damage on hit. Space law has 4.

This is a needed buff to Librarian, IAA, and Magistrate. :book:

If you're on help intent you won't damage the recipient and only educate them. follow SOP and put your clown shoes back on already

🆑
add: Books now deal light damage on hit if you're not on help intent. Space law book deals extra damage.
/🆑

### CODE REVIEW

**Comment by KasparoVy** [comment]
Does space law get a force buff when thrown for the sake of the saying '_throwing the book at them_'?

**Comment by datlo** [comment]
good call fixing that right now

**Comment by Fox-McCloud** [comment]
Bashing people with books to "educate" them is one of the funnier things to do in SS13 that is not harmful and gets a point across without having any actual repercussions.

Keeping books no harmful is more humorous than them causing harm, ergo my 👎

**Comment by datlo** [comment]
🤔 you're probably correct

Let me see if I can code something clean to solve that

**Comment by KasparoVy** [comment]
Help vs harm?

File: code/modules/library/lib_items.dm [hunk 1]
```diff
@@ -283,6 +285,14 @@
 	else
 		return ..()
 
+/obj/item/book/attack(mob/M, mob/living/user)
+	if(user.a_intent == INTENT_HELP)
+		force = 0
+		attack_verb = list("educated")
+	else
+		force = initial(force)
+		attack_verb = list("bashed", "whacked")
```
[the remark this answers is not in the document]
Comment (datlo) [reply code/modules/library/lib_items.dm]: yeah i tried that didn't work because its a list

**Comment by Nicaragua1** [comment]
Im not sure about it... I always enoyed seeing sec officers throw a space law book into someone's  cell so they can learn their crimes...If the space law does damage then they wont do it anymore because they can break the windows or attack themself/assault

File: code/modules/library/lib_items.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (KasparoVy) [reply code/modules/library/lib_items.dm]: Fair enough, my bad- forgot that didn't worj

**Comment by joniroq** [comment]
I'm all for this, and look forward to the day sec beat an antagonist into submission with a space law book....albeit very slowly

**Comment by variableundefined** [comment]
This is quite funny, but I think Fox has a point that this should remain physically harmless (even for throwing) so that book throwing retains its comedy value (instead of becoming a genuinely damaging assault)

**Comment by datlo** [comment]
Removed throwforce on books so you can still throw space law at the clown without getting permabrigged for assault

**Comment by Denthamos** [comment]
Keep in mind this would also effect the chaplains bible as it is a book subtype, which they consistently use to bash people on the heads with for healing/making go dumb in the head.

**Comment by datlo** [comment]
I actually checked that before doing this PR, the bible is not a book subtype and is not affected.

**Comment by variableundefined** [comment]
I support the new implementation

**Comment by FreeStylaLT** [comment]
I'm personally skeptical of books being lethal at _all_, but I find the compromise with help intent to be perfectly justifiable.

**Pull request merged**